R/data.table.R

```diff
@@ -2554,8 +2554,17 @@ setnames <- function(x,old,new) {
   invisible(x)
 }
 
-setcolorder <- function(x, neworder)
+setcolorder <- function(x, neworder=NULL)
```

Comment:
any reason why we change default to NULL? we can use `missing(neworder)` and argument as is. Or, what I would prefer, is to use `neworder=key(x)` but then we are not using columns that have indices. Then behavior is very clear from manual or `args(setcolorder)`.